src/Components/Accordion/Accordion.scss

```diff
@@ -0,0 +1,152 @@
+@import '../../Styles/variables';
+
+.cf-accordion {
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  margin-bottom: 2px;
```

Comment:
This should be relative to one of our margin variables.

Reply:
done!